charts/gatekeeper/values.yaml

```diff
@@ -162,7 +162,6 @@ controllerManager:
   nodeSelector: {kubernetes.io/os: linux}
   resources:
     limits:
-      cpu: 1000m
```

Comment:
we can revert the changes to this file. next time a release is cut, this will be updated automatically

Reply:
Thanks for looking at this @sozercan 👀 🙏🏼 ; done!